RELEASE.md

```diff
@@ -124,3 +158,20 @@ $ version=$(< VERSION)
 $ git tag -s "mimir-${version}" -m "v${version}"
 $ git push origin "mimir-${version}"
 ```
+
+### Cherry-picking changes into release branch
+
+To cherry-pick a change (commit) from `main` into release branch, please do the following:
+
+```bash
+$ git checkout release-X.Y                   # Start with the release branch
+$ git checkout -b cherry-pick-pr-ZZZ         # Create new branch for cherry-picking
+$ git cherry-pick -x <commit ID>             # Cherry pick the change using -x option to add original commit ID to the message
+$ git push origin cherry-pick-pr-ZZZ         # Push branch to Github.
+```
+
+After pushing branch to PR, you can create the branch by opening `https://github.com/grafana/mimir/pull/new/cherry-pick-pr-ZZZ`.
```

Comment:
I think this sentence needs some rewording. I think you mean "After pushing branch to github, you can create a PR against the release by opening..."

Reply:
Good catch! Thanks.

Reply:
You still have "after pushing branch to PR, you can create the PR"...

Reply:
🤦 Thanks!

Reply:
Hopefully fixed now :) PTAL.